lib/Dialect/TritonGPU/Transforms/Pipeliner/MatmulLoopPipeline.cpp

```diff
@@ -1080,80 +1080,54 @@ static std::optional<int> dotCanBeProperlyAsync(ttng::DotAsyncOp dotOp,
 }
 
 // If necessary, insert a dot-wait inside the loop, waiting for the results of
-// the async dots from iteration i-1 to complete.  (We pipeline to depth 2, so
-// there are at most 2 copies of each dot_async in flight at a time.)
+// the properly-async dots from iteration i-1 to complete.  (We pipeline to
```

Comment:
I have never saw this comment. This does not seem to be correct, right? @ThomasRaoux 
(It is not really related to this PR)

Reply:
what's not correct about this?

Reply:
I think it is fine, @pawelszczerbuk note that this is done as a post-processing on the pipelining

Reply:
Oh, OK, I got confused by it saying that we pipeline to the depth of 2, while in the main pipeliner I think this really depends on the num_stages

Reply:
right, for the post processing we hard code to 2 stages for the async mma copy (that's why we need to add an extra buffer for mmav3)